packages/react-components/react-toast/src/state/vanilla/createToaster.ts

```diff
@@ -131,8 +136,9 @@ export function createToaster(options: Partial<ToasterOptions>) {
       content,
       updateId: 0,
       toasterId,
-      dispatchedAt: Date.now(),
+      dispatchedAt: timestamp++,
```

Comment:
This does not seem right (`dispatchedAt` uses some random number)

Reply:
The alternative is to introduce a new field for this counter, WDYT?

Reply:
New field sounds okay if `dispatchedAt` will be removed

Reply:
Changed `dispatchedAt` to `order`